### DISCUSSION TITLE
How can I disable the creation of gh-pages?

### DISCUSSION THREAD

**Discussion by pkyeck** [upptime/upptime#215] [Q&A] [answered]
I'm using a private repo for this and don't want to use the gh-pages (status-website) feature. I deleted the gh-pages branch manually but it is re-created every time I change something and the Setup CI action runs.

**Comment by AnandChowdhary** [comment, marked as answer]
If you're using a private repository, the status website is not published by default (https://github.com/upptime/upptime/discussions/107). Only if you set `publish: true` will the site be built and published. If that's not the case only a static `index.html` saying `404` page is published.

Does this solve your use case? Or do you want an additional option to completely disable the `gh-pages` branches altogether? That's a little harder because we don't distinguish between publish and private repositories when we use the publishing workflow.

**Reply by pkyeck** [reply]
It's not perfect but working for now. Thanks
